### DISCUSSION TITLE
升级到58版本，Dialog样式无效

### DISCUSSION THREAD

**Discussion by wjwjwjw** [alibaba/formily#1522] [Q&A 2.x] [answered]
由47版本升级到58版本后dialog不居中，样式不生效了
![image](https://user-images.githubusercontent.com/33014303/119928704-aea20300-bfae-11eb-97bb-3a9949ac8ecc.png)

**Comment by janryWang** [comment, marked as answer]
需要自己手动依赖antd样式

**Reply by wjwjwjw** [reply]
为什么要做这样的修改呢，是从哪个版本修改的

**Comment by janryWang** [comment]
https://github.com/alibaba/formily/discussions/1501
